src/commands/mod.rs

```diff
@@ -688,7 +690,18 @@ impl DaemonControl {
         let final_tx = spend_psbt.extract_tx_unchecked_fee_rate();
         self.bitcoin
             .broadcast_tx(&final_tx)
-            .map_err(CommandError::TxBroadcast)
+            .map_err(CommandError::TxBroadcast)?;
+
+        // Finally, update our state with the changes from this transaction.
+        let (tx, rx) = mpsc::channel();
```

Comment:
GUI will use commands in different threads. It would require a `SyncSender`

```suggestion
        let (tx, rx) = mpsc::sync_channel();
```

Reply:
Done.